tensorflow/core/kernels/mkl/mkl_quantized_conv_ops_test.cc

```diff
@@ -178,12 +155,8 @@ class QuantizedConv2DTest : public OpsTestBase {
     const Tensor& output = *GetOutput(0);
     if (!NativeFormatEnabled()) {
       const Tensor& output_mkl_metadata = *GetOutput(3);
-
-      ConvMklToTF conv_comp;
-      Tensor output_quantized;
-      conv_comp.ConvertMklToTF<qint32>(DT_QINT32, output, output_mkl_metadata,
-                                       output_quantized);
-
+      Tensor output_quantized =
+          GetTFFormatTensor<qint32>(DT_QINT32, output, &output_mkl_metadata);
```

Comment:
Nit: Can we make `output_mkl_metadata` a `const Tensor*` instead? Same for lines 422, 510, 566, and 625.
It'd be great if we could rename all of them (including the ones that call `QuantizedToFloatTFFormat`) to `output_mkl_metadata_ptr`.

Reply:
Sure. Done.